.travis.yml

```diff
@@ -1,3 +1,8 @@
+sudo: required
```

Comment:
> sudo isn't required for pip, it's required for docker

Ah ok, that's fair enough then. Were you ok with continuing this under travis or was circle-ci a better bet? I thought it had a bit better docker support, but things may have changed since then.